sweepbatcher/sweep_batcher.go

```diff
@@ -265,6 +284,17 @@ func WithNoBumping() BatcherOption {
 	}
 }
 
+// WithSignMuSig2 instructs sweepbatcher to use a custom function to
+// produce MuSig2 signatures. If it is set, it is used to create
+// musig2 signatures instead of musig2SignSweep and signerClient. Note
+// that musig2SignSweep must be nil in this case, however signerClient
+// must still be provided, as it is used for non-coop spendings.
+func WithSignMuSig2(signMuSig2 SignMuSig2) BatcherOption {
```

Comment:
nit: consider calling this `WithCustomMuSig2Signer`.

Reply:
Fixed